src/assets/i18n/de.json

```diff
@@ -33,6 +33,7 @@
       }
     },
     "EDIT": {
+      "LABEL": "Neuigkeit bearbeiten",
```

Comment:
Duplication?
Why you added a new item here, with same content as the following line?